### ISSUE TITLE
How to use with Select options?

### ISSUE THREAD

**Issue by Lahori-Jawan** [NeXTs/Jets.js#25]
Hi. I am now adding a feature i.e. filter to my tables. What I want is to filter the data based on some given options in the form of `<select>` tag. But I don't know how. Just for the details:

`<div class="row">`

  `<div class="column">`
     `<input type="text" **id="jetsSearch"** placeholder="Search">`
   `</div>`

  `<div class="column">`
      `<select class="ui compact selection dropdown">`
                    `<option selected="" value="all"  **id="jetsSearch"**>All</option>`
                    `<option  value="articles">10</option>`
                    `<option value="products">9</option>`
         `</select>`
  `</div>`

`</div>`

 `<table class="ui celled striped table top-space">`
             
              `<tbody **id="jetsContent"** style="text-align:center;">`
                `<tr v-for="student in studentList">`
                  `<td class="colspan=3">`
                     `{{ student.name }}`
                  `</td>`
                  `<td class="colspan=3">`
                     `{{ student.fathername }}`
                  `</td>`
                  `<td class="colspan=3">`
                    ` {{ student.class }}`
                  `</td>`
                `</tr>`
             ` </tbody>`
            </table>

if I select 9 then table should show all data that contains 9 only. filtering with `input `tag is working fine but  Is it possible in `Select` tag or some other way.

**Comment by NeXTs** [comment]
wow, I didn't think about other fields rather than input :\

in theory it should work by default because it listens for change event, but we should check

would be able to investigate it later today, few hours later

**Comment by NeXTs** [comment]
yep, and it does ! :D 

http://codepen.io/NeXTs/pen/QKgrvb

**Comment by NeXTs** [comment]
I should definitely mention about it in docs, a bit later :)

**Comment by Lahori-Jawan** [comment]
Well, that's great, but codepen example has `select `tag only. In my case I have two tags to search from against one content i.e. `input ` and `select` and select is now working :/

**Comment by NeXTs** [comment]
sould like it relates to #18 

can't say when it will be implemented, I'm pretty busy at this moment :\

**Comment by Lahori-Jawan** [comment]
yeah. I just checked on fiddle https://jsfiddle.net/1ea8ohcd/ that it doesn't work for more than one inputs

**Comment by seokirill** [comment]
@Lahori-Jawan 
You have duplicate attribute ID(the same values in two elements).
It is wrong.

**Comment by NeXTs** [comment]
That's because he needs two input elements. I'll handle this case soon

пятница, 14 октября 2016 г. пользователь Kirill написал:
